contents/teams/support/objectives.mdx

```diff
@@ -1,21 +1,29 @@
+- **What we'll ship:** Enhancements to the support Max AI.
+- **We'll know we're successful when:** A larger percentage of Max's community responses are marked as solution.
 
-### Optimize Max AI in the sidebar (Steven)
-- **Rationale:** Better Max = more case deflection = more time for support to spend on more complex tickets.
-- **What we'll ship:** Optimized Max (details coming soon to a GitHub issue near you).
-- **We'll know we're successful when:** Max is regularly deflecting a good number of tickets and that number is increasing with optimizations.
+### Deflect more tickets using Max AI (Ben)
+- **Rationale:** Encouraging customers to use Max AI will decrease the number of support tickets, allowing us to focus time and effort on more complicated tickets.
+- **What we'll ship:** Improvements to the way Max is used in the support sidebar so that more customers are directed to use Max AI and/or their questions are run through Max AI as a precursor to sending the ticket to support.
+- **We'll know we're successful when:** The support contact rate is decreasing.
 
-### Everyone in support is an SME (Abigail)
-- **Rationale:** SMEs = less escalations to engineering = we ship more things.
-- **What we'll ship:** Framework and resources for developing SMEs in selected product areas. Implement these and train the current team as SMEs.
-- **We'll know we're successful when:** The percentage of escalated tickets is decreasing in areas that we have trained SMEs.
+### Improve how we search and retrieve information from internal knowledge sources (Haynes)
+- **Rationale:** Quicker and more accurate access to internal knowledge will help us respond to tickets faster and more effectively, improving both response time and resolution rates.
+- **What we'll ship:** Improved search functionality or workflows for accessing internal documentation, GitHub issues/PRs, and past tickets
+- **We'll know we're successful when:** Support team members can consistently find the information they need without spending time digging through multiple sources. First response time (FRT) should be trending down.
 
+### Create SDK example apps which can be used internally by the support team (Joshua)
+- **Rationale:** Having SDK example apps available to quickly get up and running will improve support response times and decrease the number of SDK/implementation ticket escalations.
+- **What we'll ship:** 2-3 example apps available in a public repo with fully-fledged examples of how PostHog can be integrated.
+- **We'll know we're successful when:** Support team members are able to use these apps to quickly reproduce customer issues and we're escalating fewer SDK/implementation tickets.
 
+### Better analytics on tickets (Luke)
+- **Rationale:** A better understanding of where what themes are coming up in tickets will allow us to understand what changes need to be made and the areas/products they need to be made in.
```

Comment:
this might read better?
```suggestion
- **Rationale:** A better understanding of where particular themes are coming up in tickets will allow us to understand what changes need to be made and the areas/products they need to be made in.
```